YMChat/src/main/java/com/yellowmessenger/ymchat/BotWebView.java

```diff
@@ -202,6 +203,19 @@ public void onCreate(Bundle savedInstanceState) {
             this.finish();
         });
         showCloseButton();
+
+        RelativeLayout constraintLayout = findViewById(R.id.parentView);
```

Comment:
shouldn't the var name be relativeLayout?

Reply:
can we use binding here instead of findViewById ?

Reply:
I will change the name.
Thats another gradle level change, we can discuss about this later,